plenum/server/consensus/replica_service.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, name: str, validators: List[str], primary_name: str,
         self._data = ConsensusSharedData(name, validators, 0)
         self._data.primary_name = primary_name
         config = getConfig()
-        stasher = StashingRouter(config.REPLICA_STASH_LIMIT)
+        stasher = StashingRouter(config.REPLICA_STASH_LIMIT, buses=[bus, network])
```

Comment:
So stasher slowly becomes "one central bus"... probably it's time to reconsider some of internal/external buses design